cupy/core/core.pyx

```diff
@@ -367,6 +367,12 @@ cdef class ndarray:
         .. seealso:: :meth:`numpy.ndarray.fill`
 
         """
+        if isinstance(value, numpy.ndarray):
+            if value.size != 1:
```

Comment:
Is it OK to accept a case when `value.shape == (1,)`? If so, please add a test case.